SampleCommand/PlatformCompatibilityTests.cs

```diff
@@ -114,8 +111,8 @@ public static void TestBadProcessFile()
         {
             var baseDirectory = Path.GetDirectoryName(SampleCommandPath)!;
 
-            AssertThrows<Win32Exception>(() => Command.Run(baseDirectory));
-            AssertThrows<Win32Exception>(() => Command.Run(Path.Combine(baseDirectory, "DOES_NOT_EXIST.exe")));
+            AssertThrows<InvalidOperationException>(() => Command.Run(baseDirectory));
```

Comment:
This was failing for all target frameworks, so I think this is a breaking change in a more recent version of .NET SDK, but I couldn't find anything that seems relevant in https://learn.microsoft.com/en-us/dotnet/core/compatibility/8.0 or https://learn.microsoft.com/en-us/dotnet/core/compatibility/7.0.